add note to CONTRIBUTING.md (#236)
temporalio/sdk-typescript: CONTRIBUTING.md

  git submodule init
  git submodule update
  ```

>   > if you get a `The authenticity of host 'github.com (192.30.252.123)' can't be established.` error, run `ssh-keyscan github.com >> ~/.ssh/known_hosts` and retry.

Reviewer: If

Reviewer 2: I've been using this for over a decade, nice to know about `ssh-keyscan` (in an alias)
```
perl -i -n -e "print unless (\$. == $1)" ~/.ssh/known_hosts <line_number>
```

Author: ```suggestion
  > If you get a `The authenticity of host 'github.com (192.30.252.123)' can't be established.` error, run `ssh-keyscan github.com >> ~/.ssh/known_hosts` and retry.
```

- Install the dependencies:
  ```sh
  npm ci
  ```

>   This is a longer process that involves downloading and compiling Rust dependencies, wait a few minutes.

Reviewer: Might be a small thing, but I try to avoid commanding the reader (eg "wait"). Perhaps "This may take a few minutes, as it involves downloading and compiling Rust dependencies."

Author: ```suggestion
  This may take a few minutes, as it involves downloading and compiling Rust dependencies.
```
